packages/slice-machine/components/Changelog/versionDetails/UpdateCommandBox.tsx

```diff
@@ -63,18 +85,40 @@ export const UpdateCommandBox: React.FC<UpdateCommandBoxProps> = ({
 
 function getInstallCommand(
   packageManager: PackageManager,
-  packageSpec: string
+  packagesSpecs: string[]
 ): string {
+  const packagesSpecsStr = packagesSpecs.join(" ");
+
   switch (packageManager) {
     case "bun":
-      return `bun add --development ${packageSpec}`;
+      return `bun add --development ${packagesSpecsStr}`;
     case "npm":
-      return `npm install --save-dev ${packageSpec}`;
+      return `npm install --save-dev ${packagesSpecsStr}`;
     case "pnpm":
     case "pnpm@6":
-      return `pnpm add --save-dev ${packageSpec}`;
+      return `pnpm add --save-dev ${packagesSpecsStr}`;
     case "yarn":
     case "yarn@berry":
-      return `yarn add --dev ${packageSpec}`;
+      return `yarn add --dev ${packagesSpecsStr}`;
   }
 }
```

Comment:
We might want to use the existing logic from the manager to retrieve the installation command: https://github.com/prismicio/slice-machine/blob/1db109d2d74159ee5abbc3ebcf6c456b9430716a/packages/manager/src/lib/installDependencies.ts#L30-L59

Note that we would need to call an async method via `managerClient` if we made that change, which may introduce some unnecessary complexity. You can be the judge on whether or not it's worth it. 🙂

Reply:
That could be a good idea for the front-end not having to know this management. My PR is already a bit too big, and it's a bit out of the score of this issue. When we will redesign this page, I could propose such a change and investigate if we would like this, ok for you?

Reply:
Yep, works for me. I definitely think we should consider it when the page is reworked, but it's not needed now.